src/vm/eeconfig.cpp

```diff
@@ -857,7 +863,7 @@ fTrackDynamicMethodDebugInfo = CLRConfig::GetConfigValue(CLRConfig::UNSUPPORTED_
 #endif // FEATURE_CONSERVATIVE_GC
 
 #ifdef _WIN64
-    iGCAllowVeryLargeObjects = (CLRConfig::GetConfigValue(CLRConfig::EXTERNAL_gcAllowVeryLargeObjects) != 0);
+    iGCAllowVeryLargeObjects = Configuration::GetKnobBooleanValue(W("System.GC.AllowVeryLargeObjects"), CLRConfig::EXTERNAL_gcAllowVeryLargeObjects);
```

Comment:
I do not think we need official switch for `AllowVeryLargeObjects`. It is left over compat quirk from desktop.